src/main/java/cz/kominekjan/disenchantment/events/ShatterEvent.java

```diff
@@ -80,10 +62,11 @@ public void onDisenchantmentEvent(PrepareAnvilEvent e) {
         HashMap<String, IPlugin> activatedPlugins = PluginManager.getActivatedPlugins();
 
         if (activatedPlugins.isEmpty()) {
-            book = VanillaPlugin.createEnchantedBook(enchantments);
+            book = VanillaPlugin.createEnchantedBook(randomEnchantmentShatter);
         } else {
             for (IPlugin plugin : activatedPlugins.values()) {
-                book = plugin.createEnchantedBook(enchantments);
+                // won't 2 plugin activated on the same time would only use the last book ?
```

Comment:
I dont think anyone would be brave enough to have more than 1 custom enchantment plugin running, cant imagine the amount of times it makes a collision or something :DDDD